src/docfx/build/xref/XrefResolver.cs

```diff
@@ -166,7 +170,7 @@ public XrefMapModel ToXrefMapModel(bool isLocalizedBuild)
                     .ToArray();
             }
 
-            var model = new XrefMapModel { References = references };
+            var model = new XrefMapModel { References = references, RepositoryUrl = _repository?.Remote };
```

Comment:
It seems strange that there is a `references.Length > 0` check on line 171, would you help remove that as well? To make the tags appear regardless of reference count just like repositoryUrl.

Reply:
OK